### PR TITLE
creater B827EBFFFE494323.json

### PR DESCRIPTION
**Pull request by viniaero**

### CODE REVIEW

File: B827EBFFFE494323.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE494323", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-brazil",
```
Comment (gonzalocasas) [inline B827EBFFFE494323.json]: This is not a valid router address, check the list of valid routers: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

File: B827EBFFFE494323.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		"gateway_ID": "B827EBFFFE494323", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-brazil", 
+				"serv_port_up": 1700, 
+				"serv_port_down": 1700, 
+				"serv_enabled": true 
+			} 
+		], 
+		"ref_latitude": the lat of the rak 831 gateway,
```
Comment (gonzalocasas) [inline B827EBFFFE494323.json]: you need to enter a decimal value here.

File: B827EBFFFE494323.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-brazil", 
+				"serv_port_up": 1700, 
+				"serv_port_down": 1700, 
+				"serv_enabled": true 
+			} 
+		], 
+		"ref_latitude": the lat of the rak 831 gateway, 
+		"ref_longitude": the long of the rak 831 gateway,
```
Comment (gonzalocasas) [inline B827EBFFFE494323.json]: you need to enter a decimal value here.

Comment (viniaero) [inline B827EBFFFE494323.json]: updated

**Pull request merged**